src/pages/index.vue

```diff
@@ -86,18 +92,32 @@ const selectDataForRoute = async () => {
     unknownProps.value = unknown;
     if (unknownProps.value.length) return;
 
+    const { pathogen, version, indicator, country, adminLevel } = propsWithCorrectCase;
     // we pick dengue, may24 and FOI as defaults for pathogen, version and indicator respectively
-    if (!props.indicator) {
+    if (!indicator) {
         router.replace(`/${APP_BASE_ROUTE}/${possibleValuesForProps.indicator[0]}`);
         return;
     }
+    if (country) {
+        const admin2DataMissing = appConfig.value.countriesWithoutAdmin2.includes(country);
+        if (!adminLevel) {
+            router.replace(
+                `/${pathogen}/${version}/${indicator}/${country}/${admin2DataMissing ? AdminLevel.ONE : AdminLevel.TWO}`
+            );
+            return;
+        }
```

Comment:
Is this worth an e2e test? (genuine question)

Reply:
yh probably worth test if a country is missing data then admin1 is in url, i dont think it is worth explicitly testing the flip side though